CHANGELOG.md

```diff
@@ -1,6 +1,16 @@
 # Change Log
 All notable changes to the "azure-appservice" extension will be documented in this file.
 
+## 0.10.0 - 2018-11-28
+### Added
+- CosmosDB Connections: Leverage CosmosDB Extension (https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-cosmosdb) to effortlessly connect databases to web apps
+- Update default deploy settings option
```

Comment:
"Update default deploy settings option" I don't even know what this means. What feature is this for?

Reply:
The button that Andrii worked on.  I'll think of better wording.